lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -101,176 +101,184 @@ $notes-annotator-background-dark: rgba(122,122,122,0.6); // taken from annotator
   border-color: $error-color;
 }
 
-// +individual note (in context)
-// --------------------
-.annotator-outer.annotator-outer {
-  @extend %ui-depth4;
-  @extend %notes-reset-font;
-}
+.edx-notes-wrapper {
+  // +individual note (in context)
+  // --------------------
+  .annotator-outer {
+    @extend %ui-depth4;
+    @extend %notes-reset-font;
+  }
 
-// bubble
-.annotator-widget.annotator-widget {
-  @extend %notes-bubble;
-}
+  // bubble
+  .annotator-widget.annotator-widget {
+    @extend %notes-bubble;
+  }
 
-.annotator-item {
-  padding: ($baseline/2) !important;
-}
+  .annotator-listing {
+    padding: 0 !important;
+    margin: 0 !important;
+  }
 
-// +creating/editing notes (overrides for vendor styling)
-// --------------------
-// adding
-.annotator-adder {
-  @extend %notes-reset-background;
+  .annotator-item {
+    padding: ($baseline/2) !important;
+    margin-bottom: 0 !important;
```

Comment:
Fair enough - its scoped well enough to not cause any more cascading/specificity trouble.